packages/shell-chrome/src/backend.js

```diff
@@ -57,16 +57,23 @@ function handleMessages(e) {
           var splittedAttrs = e.data.payload.attributeSequence.split("*");
 
           for (index in splittedAttrs) {
```

Comment:
Can this for loop be moved inside of the if statement on line 66/67?

Reply:
Lol, yes it can https://github.com/Te7a-Houdini/alpinejs-devtools/pull/23

I tried to keep this PR low(ish) in scope

Reply:
Funny to say "PR low in scope" when I added support for number input instead of only boolean 😂